src/common/options.cc

```diff
@@ -5007,7 +5007,7 @@ std::vector<Option> get_rgw_options() {
     .set_default(0)
     .set_description("The maximum length of metadata value. 0 skips the check"),
 
-    Option("rgw_max_attr_name_len", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    Option("rgw_max_attr_name_len", Option::TYPE_SIZE, Option::LEVEL_ADVANCED)
```

Comment:
it's the length of rgw attribute name, so i don't think it's a size per se.

Reply:
it's measuring characters/bytes? length of a string?  seems like a size to me

Reply:
it's measuring number of characters.